exonum/src/node/state.rs

```diff
@@ -549,6 +549,11 @@ impl State {
         &self.peers
     }
 
+    /// Returns the addresses of known connections with their public keys.
```

Comment:
Comment is not clear, because public key identifies validator, not the connection.